Index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+       
+  <title>How to write well-formed XHTML</title>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+  <script src="js/Ass.js"></script>
+  <meta charset="utf-8"> 
+</head>
+<body>
+    <!-- Mahougbe C. Houffon
+     assignment2.htm
+     INFO1311.WW
+     Lisa THOENDEL 
+     12/04/19
+-->
+    <h1>Greetings</h1>
+<p> Hello, my name is <span id="first-name"></span> <span id="last-name"></span></p>
+<p>I am a student at <span class="mcc"> Metropolitan Community College</span> in Omaha, Nebraska!</p>
+<p>My hobbies Include:<br>
+<ul>
+    <li id="Hobby"> Cooking</li>
+    <li id="Hobby"> Soccer</li>
+    <li id="Hobby"> Travel</li>
```

Comment:
So these list items should be just like the one in the figure in the assignment - "List hobby here" and the values should be added in the .js as we talked about. You did a great job with those name spans and the mcc class!